elfi/results/result.py

```diff
@@ -106,8 +107,9 @@ def summary(self):
         if hasattr(self, 'threshold'):
             desc += "Threshold: {:.3g}\n".format(self.threshold)
         print(desc, end='')
-        self.posterior_means()
+        self.posterior_means
 
+    @property
     def posterior_means(self):
         """Print a representation of posterior means.
```

Comment:
I think it would be better to return these as arrays. Then these would be usable to other things also besides printing.